entity/libraryerror.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# This class for book
+class LibraryError < StandardError
+end
```

Comment:
file naming `library_error.rb`

And better to use different errors for different checks for example (LibraryClassError) for invalid class or StringError for string which empty etc..

Reply:
StringError and LibraryClassError - I need to add this class from LibraryError ?